manager/install.sh

```diff
@@ -23,64 +23,218 @@ mkdir /workspace
+  echo "✓"
+
+  echo -n "￮ configuring networking "
+  setup_istio
+  python render_template.py $CORTEX_CLUSTER_CONFIG_FILE manifests/apis.yaml.j2 > /workspace/apis.yaml
+  kubectl apply -f /workspace/apis.yaml >/dev/null
+  echo "✓"
+
+  echo -n "￮ configuring autoscaling "
+  python render_template.py $CORTEX_CLUSTER_CONFIG_FILE manifests/cluster-autoscaler.yaml.j2 > /workspace/cluster-autoscaler.yaml
+  kubectl apply -f /workspace/cluster-autoscaler.yaml >/dev/null
+  echo "✓"
+
+  echo -n "￮ configuring logging "
+  envsubst < manifests/fluentd.yaml | kubectl apply -f - >/dev/null
+  echo "✓"
+
+  echo -n "￮ configuring metrics "
+  envsubst < manifests/metrics-server.yaml | kubectl apply -f - >/dev/null
+  envsubst < manifests/statsd.yaml | kubectl apply -f - >/dev/null
+  echo "✓"
+
+  if [[ "$CORTEX_INSTANCE_TYPE" == p* ]] || [[ "$CORTEX_INSTANCE_TYPE" == g* ]]; then
+    echo -n "￮ configuring gpu support "
+    envsubst < manifests/nvidia.yaml | kubectl apply -f - >/dev/null
+    echo "✓"
+  fi
+
+  if [[ "$CORTEX_INSTANCE_TYPE" == inf* ]]; then
+    echo -n "￮ configuring inf support "
+    envsubst < manifests/inferentia.yaml | kubectl apply -f - >/dev/null
+    echo "✓"
+  fi
+
+  if [ "$CORTEX_API_LOAD_BALANCER_SCHEME" == "internal" ] && [ "$CORTEX_API_GATEWAY" == "public" ]; then
+    create_vpc_link_integration
+  fi
+
+  restart_operator
+
+  validate_cortex
+
+  await_pre_download_images
+
+  if [ "$CORTEX_OPERATOR_LOAD_BALANCER_SCHEME" == "internal" ]; then
+    echo -e "\ncortex is ready! (it may take a few minutes for your private operator load balancer to finish initializing, but you may now set up VPC Peering)"
+  else
+    echo -e "\ncortex is ready!"
+  fi
+}
+
+function cluster_configure() {
+  check_eks
+
+  resize_nodegroup
+
+  echo -n "￮ updating cluster configuration "
+  setup_configmap
+  setup_secrets
+  echo "✓"
+
+  restart_operator
+
+  validate_cortex
+
+  echo -e "\ncortex is ready!"
+}
+
+# creates the eks cluster and configures kubectl
+function create_eks() {
   set +e
   cluster_info=$(eksctl get cluster --name=$CORTEX_CLUSTER_NAME --region=$CORTEX_REGION -o json 2> /dev/null)
   cluster_info_exit_code=$?
   set -e
 
-  # No cluster
-  if [ $cluster_info_exit_code -ne 0 ]; then
-    if [ "$arg1" = "--update" ]; then
-      echo "error: there is no cluster named \"$CORTEX_CLUSTER_NAME\" in $CORTEX_REGION; please update your configuration to point to an existing cortex cluster or create a cortex cluster with \`cortex cluster up\`"
+  # cluster already exists
+  if [ $cluster_info_exit_code -eq 0 ]; then
+    set +e
+    # cluster statuses: https://github.com/aws/aws-sdk-go/blob/master/service/eks/api.go#L6883
+    cluster_status=$(echo "$cluster_info" | jq -r 'first | .Status')
+    set -e
+
+    if [ "$cluster_status" == "ACTIVE" ]; then
+      echo "error: there is already a cluster named \"$CORTEX_CLUSTER_NAME\" in $CORTEX_REGION"
+      exit 1
+    elif [ "$cluster_status" == "DELETING" ]; then
```

Comment:
Do you think that this code is still necessary since we check the cluster status in golang before we run the manager?